app-emulation/xen-tools/xen-tools-4.16.0-r3.ebuild

```diff
@@ -69,7 +69,7 @@ SLOT="0/$(ver_cut 1-2)"
 # Inclusion of IUSE ocaml on stabalizing requires maintainer of ocaml to (get off his hands and) make
```

Comment:
Needs revbump (git mv is fine) which I'll do.

Reply:
Thanks @thesamesam